src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -0,0 +1,54 @@
+import { ReactNode, FC } from 'react';
+import {
+  createTheme,
+  ThemeProvider as MuiThemeProvider,
+} from '@mui/material/styles';
+
+interface Props {
+  children: ReactNode;
+}
+
+const ThemeProvider: FC<Props> = function ThemeProvider({ children }) {
+  const theme = createTheme({
+    breakpoints: {
+      values: {
```

Comment:
Do yo need these overrides of breakpoints and typography?

Reply:
Now I removed some overrides of typography and left only useful. 
About breakpoints - I need only 2 breakpoints to override, but it is a requirement of MUI to define all breakpoints - 'If you change the default breakpoints's values, you need to provide them all'.

Reply:
Ok. Add comment about it, please